src/components/news-page/news-content-group.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './news-content-group.styles.scss';
+
+const NewsContentGroup = (props) => (
+    <div className="news-content-group-background">
+        <div className="two-sides-container">
+            <div className="image-container">
+                <img className="image" src={props.image} alt="img" />
+            </div>
+            <div className="text-group-container">
+                <div className="text-group">
+                    <div className="article">{props.article}</div>
+                    <div className="bottom-text-group">
+                        <div className="date">{props.date}</div>
+                        <div className="link">{props.link}</div>
```

Comment:
`link` could get confused  for a url if another developer tried to use this component